index.bs

```diff
@@ -334,7 +334,7 @@ if (cookie.secure)
 Both [=documents=] and [=service workers=] access the same modification API, via the
 {{Window/cookieStore}} property on the [[#globals|global object]].
 
-Cookies are created or modified (written) using the {{CookieStore/set(name, value, options)|set()}} method.
+Cookies are created or modified (written) using the {{CookieStore/set(options)|set()}} method.
```

Comment:
Given that the following example uses name, value shouldn't this be {{CookieStore/set(options)|set()}} ?

Reply:
Ah you are right, thanks for the catch.